modules/publication/ajax/FileUpload.php

```diff
@@ -865,15 +873,28 @@ function editUploads($id)
  * Utility function to return errors from the server
  *
  * @param string $message error message to display
+ * @param int    $code    HTTP response code
  *
  * @return void
  */
-function showError($message)
+function showError($message, $code = 500)
 {
     if (!isset($message)) {
         $message = 'An unknown error occurred!';
     }
-    header('HTTP/1.1 500 Internal Server Error');
+    $resp = '';
```

Comment:
If you want a shortcut here you can just use

> [http_response_code](https://secure.php.net/manual/en/function.http-response-code.php) ($code); 

and delete everything until the Content-Type header.